### PR TITLE
Fix infinite loop when query panel is closed.

### PR DESCRIPTION
**Pull request by offtherailz**
This error happens when closing the query panel (in debug mode) .

image
This is due to the onClick event bind directly to the action, passing the wrong parameters (the mouseEvent instead of the search URL). 
In debug this causes an immutable error, in general the state becomes not consistent anymore.

### CODE REVIEW

File: web/client/plugins/Map.jsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ const MapPlugin = React.createClass({
     },
     getDefaultProps() {
         return {
-            mapType: 'leaflet',
+            mapType: 'cesium',
```
Comment (mbarto) [inline web/client/plugins/Map.jsx]: why?

Comment (offtherailz) [inline web/client/plugins/Map.jsx]: didn't wanted to commit that. sorry, I'm going to revert this change
